src/libs/kata-types/src/annotations/mod.rs

```diff
@@ -271,6 +271,9 @@ pub const KATA_ANNO_CFG_HYPERVISOR_VIRTIO_FS_EXTRA_ARGS: &str =
     "io.katacontainers.config.hypervisor.virtio_fs_extra_args";
 /// A sandbox annotation to specify as the msize for 9p shares.
 pub const KATA_ANNO_CFG_HYPERVISOR_MSIZE_9P: &str = "io.katacontainers.config.hypervisor.msize_9p";
+/// A sandbox annotation to support Intel tdx quote generation service socket port(only supported in QEMU currently).
+pub const KATA_ANNO_CFG_HYPERVISOR_QGS_SOCKET_PORT: &str =
+    "io.katacontainers.config.hypervisor.qgs_socket_port";
```

Comment:
this is potentially problematic setting because users don't really know how the cluster admins have configured QGS. I'd suggest to drop this annotation completely.

Reply:
Yes you're right, i will update the PR without this annotation